apps/aechannel/src/aesc_offchain_update.erl

```diff
@@ -63,14 +63,16 @@ op_new_contract(OwnerId, VmVersion, Code, Deposit, CallData) ->
 
 
 -spec op_call_contract(aec_id:id(), aec_id:id(), aect_contracts:vm_version(),
-                       non_neg_integer(), aect_call:call(), [non_neg_integer()]) -> update().
+                       non_neg_integer(), Call, [non_neg_integer()]) -> update()
+    when Call :: _.
```

Comment:
Unclear if https://github.com/aeternity/aeternity/blob/9d4e2f704ea9872ba16f4c2052c020e6ca44e8d0/apps/aechannel/src/aesc_offchain_update.erl#L15-L16 requires update too.

Following the code it seems it shall be `binary()`: https://github.com/aeternity/aeternity/blob/9d4e2f704ea9872ba16f4c2052c020e6ca44e8d0/apps/aechannel/src/aesc_offchain_update.erl#L117-L118

@uwiger ?